examples/seq2seq/seq2seq_trainer.py

```diff
@@ -189,7 +189,8 @@ def prediction_step(
         }
 
         if self.args.predict_with_generate and not self.args.prediction_loss_only:
-            generated_tokens = model.generate(
+            _model = model.module if hasattr(model, "module") else model
+            generated_tokens = _model.generate(
```

Comment:
Normally the `model` attribute of the `Trainer` is always a reference to the real model (without the `module` from `DataParallel` and the likes), so using `self.model` here should prevent this error.